js/src/components/experience-rating-banner/index.js

```diff
@@ -13,25 +13,84 @@ import { store as preferencesStore } from '@wordpress/preferences';
  */
 import AppButton from '../app-button';
 import FeedbackModal from './feedback-modal';
+import { recordGlaEvent } from '~/utils/tracks';
 import usePreference from '~/hooks/usePreference';
-import { PREFERENCES_STORE_NAMESPACE } from '~/constants';
+import {
+	APP_RATINGS_BANNER_CONTEXT,
+	PREFERENCES_STORE_NAMESPACE,
+} from '~/constants';
 import './index.scss';
 
 const BANNER_DISMISSED_KEY = 'experience-rating-banner-dismissed';
 
+/**
+ * @event gla_app_ratings_shown
```

Comment:
@mukeshpanchal27 A short description explaining when the events are fired please.